includes/API/Events.php

```diff
@@ -120,25 +126,13 @@ public function create_item( $request ) {
 
 		// If request isn't to be queued, we want the realtime response.
 		if ( ! $request['queue'] ) {
-			$notifications  = array();
-			$hiive_response = $this->hiive->notify( array( $event ), true );
-
-			if ( is_wp_error( $hiive_response ) ) {
-				return new \WP_REST_Response( $hiive_response->get_error_message(), 401 );
-			}
-
-			$status_code = wp_remote_retrieve_response_code( $hiive_response );
-
-			if ( 200 !== $status_code ) {
-				return new \WP_REST_Response( wp_remote_retrieve_response_message( $hiive_response ), $status_code );
-			}
+			$hiive_response_notifications = $this->hiive->send_event( $event );
 
-			$payload = json_decode( wp_remote_retrieve_body( $hiive_response ) );
-			if ( $payload && is_array( $payload->data ) ) {
-				$notifications = $payload;
+			if ( is_wp_error( $hiive_response_notifications ) ) {
+				return new \WP_REST_Response( $hiive_response_notifications->get_error_message(), $hiive_response_notifications->get_error_code() );
```

Comment:
@BrianHenryIE We'll want to return an explicit HTTP status code here. The WP_Error `get_error_code()` method isn't a status code. It would be something like `invalid_rest_request` or some other string.

Reply:
I think it will be appropriate to return 500 between the browser and the WP plugin.  The browser doesn't really need to know about 401 etc.